File: ~2020.11.26 - codingsample/202008/20200825/Through_the_Fog_1_circleOfNumbers.java
package N7_Through_the_Fog;

public class Through_the_Fog_1_circleOfNumbers {
	static int circleOfNumbers(int n, int firstNumber) {
	    return (firstNumber+(n/2))%n;
	}
	public static void main(String[] args) {
		System.out.println(circleOfNumbers(10,2));
	}

}

File: etc/~2020.11.26 - codingsample/202008/20200825/Through_the_Fog_1_circleOfNumbers.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>
#include <string.h> 

int circleOfNumbers(int n, int firstNumber) {
    return (firstNumber+(n/2))%n;
}


int main() {
	printf("%d\n",circleOfNumbers(10,2));
}


